apps/ledger-live-mobile/src/screens/Portfolio/index.tsx

```diff
@@ -99,7 +99,7 @@ function PortfolioScreen({ navigation }: NavigationProps) {
 
   const { isDrawerOpen, openDrawer, closeDrawer } = useModularDrawer();
 
-  const currencies = listAndFilterCurrencies({ includeTokens: true });
+  const currencies = useMemo(() => listAndFilterCurrencies({ includeTokens: true }), []);
```

Comment:
you could just lift `const currencies = listAndFilterCurrencies({ includeTokens: true });` out of the component rather than using useMemo